### PR TITLE
Add dimensions to cluster alarm metrics

### PR DESCRIPTION
**Pull request by elemoine**

### CODE REVIEW

**Comment by elemoine** [comment]
Thanks for the review @simonpasquier. Before merging this I'd like to modify the haproxy configuration in the user model (add `service_name`'s) and do some testing.

File: haproxy/meta/heka.yml [hunk 1]
```diff
@@ -99,6 +99,10 @@ aggregator:
 {%- if listen.get('type', None) == 'openstack-service' %}
       - {{ listen_name }}_http_errors
 {%- endif %}
+{%- if listen.service_name is defined %}
+      dimension:
+        service: {{ listen.service_name }}
```
Comment (cznewt) [inline haproxy/meta/heka.yml]: can you fallback to {{ listen.get('service_name', listen_name) }} ? does it make sense here?

Comment (elemoine) [inline haproxy/meta/heka.yml]: Yes it does, I'll change this and rebase.

Comment (elemoine) [inline haproxy/meta/heka.yml]: Done.

**Pull request merged**